pkg/utils/poddisruptionbudget.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2022 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"math"
+
+	druidv1alpha1 "github.com/gardener/etcd-druid/api/v1alpha1"
+)
+
+// CalculatePDBminAvailable calculates the minimum available value for the PDB
+func CalculatePDBminAvailable(etcd *druidv1alpha1.Etcd) int {
+	// do not enable in single mode
+	if etcd.Spec.Replicas < 2 {
+		return 0
+	}
+
+	// cluster is in bootstrap
+	if etcd.Status.ClusterSize == nil {
```

Comment:
Is this check sufficient to check if the cluster is in bootstrap stage?

Reply:
I think so, yes
Custodian controller will take over after that and update the `minAvailable`